aiidalab_widgets_base/structures.py

```diff
@@ -221,7 +221,7 @@ def store_structure(self, _=None):
         ):
             # Make a link between self.input_structure and self.structure_node
             @engine.calcfunction
-            def user_modifications(_source_structure):
+            def user_modifications(source_structure):  # noqa
```

Comment:
It would be better to have a more specific noqa, for the specific linter rule that ruff is complaining about. Please have a look at their docs, I don't remember the notation from the top of my head.